corehq/util/test_utils.py

```diff
@@ -54,3 +55,17 @@ def get_json(cls, name, override_path=None):
     @classmethod
     def get_xml(cls, name, override_path=None):
         return cls.get_file(name, 'xml', override_path)
+
+
+def flag_enabled(toggle_class):
+    """
+    Decorate test methods with this to mock the lookup
+
+        @flag_enabled(toggles.MULTIPLE_LOCATIONS_PER_USER)
+        def test_something_fancy(self):
+            something.which_depends(on.MULTIPLE_LOCATIONS_PER_USER)
+    """
+    return mock.patch(
+        '.'.join([toggle_class.__module__, toggle_class.__class__.__name__, 'enabled']),
+        new=lambda *args: True,
+    )
```

Comment:
I'm mocking out `toggles.MULTIPLE_LOCATIONS_PER_USER.enabled`, which is a function, so it needs to return a callable.  Alternatively, I think I could've specified `return_value` or whatever, but some combinations of parameters pass the mock object in to the decorated method, which would be very confusing.

Honestly, any time I try mocking something, I pretty much just guess and check until it does what I wanted it to (which is why I bothered to abstract this - so I don't have to figure it out again in the future).  Here I mostly copy-pasted from [the last time I did this sort of mock](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/tests/util.py#L118-L126).